starter_code/css/mobile.css

```diff
@@ -0,0 +1,198 @@
+
+
+@media (min-width: 320px){
+	/*small portrait screens*/
+
+
+.container{
+	max-width: 300px;
+}
+
+.logo {
+  font-size: 30px;
+  font-weight: 800;
+}
+
+
+#hero {
+  height: auto;
+  padding-top: 12px;
+  padding-bottom: 12px;
+}
+
+#hero h2{
+	font-size: 24px;
+}
+
+#benefits {
+  padding: 20px 0 5px;
+}
+
+h3 {
+  color: #033048;
+  font-size: 25px;
+  font-weight: 600;
+  margin-bottom: 10px;
+}
+
+
+nav {
+  float: right;
```

Comment:
Personally I probably would have saved anything that applies multi-column layout to a larger screen sizes like tablet or desktop